### PR TITLE
Showing which BTN/KB modifier used on CommandBar example

### PR DESCRIPTION
**Pull request by salmanmkc**
Addressing main aspect of https://github.com/microsoft/Xaml-Controls-Gallery/issues/301

## Description
https://github.com/microsoft/Xaml-Controls-Gallery/issues/301

## Motivation and Context
Showing what button/keyboard modifier was used last.

## How Has This Been Tested?
Compiling, pressing the buttons and using the keyboard modifiers.

## Screenshots (if appropriate):
Before (nothing happens):
image

After:
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: XamlControlsGallery/ControlPages/CommandBarPage.xaml [hunk 1]
```diff
@@ -44,21 +44,22 @@
     </Page.Resources>
     <StackPanel>
         <local:ControlExample x:Name="Example3" HeaderText="A command bar with labels on the side free floating in a page" >
+            <StackPanel>
+                <TextBlock x:Name="SelectedOptionText"></TextBlock>
```
Comment (stmoy) [inline XamlControlsGallery/ControlPages/CommandBarPage.xaml]: I agree with @chingucoding. Other samples (like CommandBarFlyout) display the selected option below the content. I think we should do the same here - please move this to below the CommandBar and consider adding more padding.

Comment (salmanmkc) [reply XamlControlsGallery/ControlPages/CommandBarPage.xaml]: Awesome, will do now

**Comment by salmanmkc** [comment]
Tried different spacing, both scale well (i.e. multiples of 4)
image
image

**Comment by salmanmkc** [comment]
Tried different spacing, both scale well (i.e. multiples of 4)
image
image

**Comment by salmanmkc** [comment]
Tried different padding, both scale well (i.e. multiples of 4) and ended up going for 8 as it fits better with the padding with the other controls on the page.
image
image

**Pull request merged**